includes/discovery/storage/CISCO-FLASH-MIB.inc.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * CISCO-FLASH-MIB.inc.php
+ *
+ * LibreNMS storage discovery module for Cisco Flash
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       https://www.librenms.org
+ * @copyright  2022 Félix Bouynot
+ * @author     Félix Bouynot <felix.bouynot@setenforce.one>
+ */
+
+use LibreNMS\Util\Number;
+
+$ciscoFlashPartitionName = snmpwalk_cache_oid($device, 'ciscoFlashPartitionName', null, 'CISCO-FLASH-MIB');
```

Comment:
Could you please add a condition here to check if we are indeed running the discovery against a Cisco Device (You can check for the group "cisco", should be set for all cisco devices).

Reply:
Thanks, there is now a check.